### PR TITLE
Variants

### PR DESCRIPTION
**Pull request by ynohat**

### CODE REVIEW

File: dockerfiles/terraform.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+#########
+
+FROM $BASE
+
+ARG TF_PLUGIN_CACHE_DIR="/var/terraform/plugins"
+ENV TF_PLUGIN_CACHE_DIR="${TF_PLUGIN_CACHE_DIR}"
+
+RUN apk add --no-cache curl ca-certificates
+
+COPY --from=builder /usr/local/bin/terraform.upx /usr/local/bin/terraform
```
Comment (javiergarza) [inline dockerfiles/terraform.Dockerfile:75]: Shouldn't the Akamai Terraform CLI package also be installed here?

File: variants [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+cli cps
+cli dns
+cli edgeworkers
+cli firewall
+cli image-manager
+cli property-manager
+cli property
+cli purge
+cli sandbox
+cli visitor-prioritization
```
Comment (javiergarza) [inline variants:42]: Do we want to add the cli terraform package https://github.com/akamai/cli-terraform or include it inside the terraform image?

Comment (ynohat) [reply variants:42]: good question! I'd forgotten about that CLI. @lukaszczerpak thoughts?

Comment (lukaszczerpak) [reply variants:42]: it makes sense to create a new image variant with terraform + terraform-cli

Comment (ynohat) [reply variants:42]: referenced in #14

**Pull request merged**